lib/daru/io/importers/rails.rb

```diff
@@ -0,0 +1,150 @@
+require 'daru/io/importers/base'
+
+module Daru
+  module IO
+    module Importers
+      class Rails < Base
+        Daru::DataFrame.register_io_module :read_rails, self
+        Daru::DataFrame.register_io_module :from_rails, self
+
+        def initialize
+          require 'request_log_analyzer'
+          RequestLogAnalyzer::Source::LogParser.class_eval do
+            def initialize(format, options = {})
```

Comment:
Better extract it all into `importers/rails_logs/request_log_analyzer_patch.rb`, create there `module RailsLogAnalyzerPatch`, and here just include that into `RailsLogAnalyzer`.

Reply:
The current `initialize`,`parse` and `read` methods are going to be the same for all the importers that will be added. Shall I instead create `RequestLogAnalyzer` in `importers/request_log_analyzer.rb`? Then I will pass `:rails3`, `:apache` and `:amazon_s3` to the module in `rails.rb`, `apache.rb` and `amazon_s3.rb`.